docs/sources/configuration/perception.rst

```diff
@@ -0,0 +1,51 @@
+Perception
+============
+
+The collision avoidance depends mostly on the Configuration > Perception dialog where the connection and description of either a laserscan sensor or an obstacle map message is configured.
+
+.. image:: ../data/cpanel5.png
+   :width: 800px
+   :alt: Configuration > Perception page: The laserscan or list of obstacles is configured here.
+
+- **Laserscan sensor**: The Supervisor can handle data points from one 2-dimensional laser (LIDAR) scanner.
+
+  * **Display name**: A user-specified name can be provided so that diagnostic information can quickly be understood.
```

Comment:
This display name is used when aggregated statistics about the scanner are uploaded to the 3Laws cloud dashboard.  This name is chosen by the user to help keep track of the scanner, e.g. "front_lidar".